app/controllers/spaces_controller.rb

```diff
@@ -86,4 +71,20 @@ def space_params
                                  :image_url, :style, :planet)
   end
 
+  def query_search_with_dates
```

Comment:
Didn't Nate mention we should make these private? No big deal, but just want to make sure.